api/db/migrations/20210908085902_add-method-column-in-assessments.js

```diff
@@ -0,0 +1,31 @@
+const TABLE_NAME = 'assessments';
+const COLUMN_NAME = 'method';
+
+exports.up = async function(knex) {
+  await knex.schema.table(TABLE_NAME, (table) => {
+    table.text(COLUMN_NAME);
+  });
+
+  const nbRows = (await knex.raw('select reltuples from pg_class where relname=\'assessments\'')).rows.shift().reltuples;
```

Comment:
Je connaissais pas c'est cool 🚀

Reply:
Le `knex.raw` ne me paraît pas justifié ici :
```suggestion
  const [ nbRows ] = await knex('pg_class').where({relname:'users'}).pluck('reltuples');
```